pkg/build/controller/build/build_controller.go

```diff
@@ -481,7 +482,7 @@ func (bc *BuildController) createPodSpec(build *buildapi.Build) (*v1.Pod, error)
 	return podSpec, nil
 }
 
-// resolvePushSecretAsReference returns a LocalObjectReference to a secret that should
+// resolveImageSecretAsReference returns a LocalObjectReference to a secret that should
```

Comment:
update the description below... push + pull

Reply:
done